spec/lib/resource_kit/testing/have_action_matchers_spec.rb

```diff
@@ -105,4 +105,53 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  describe '#failure_message' do
+    let(:resource_class) {
+      Class.new(ResourceKit::Resource).tap { |resource|
```

Comment:
With `Class.new()` it actually yields a block in the context of the newly created class. So no need for `tap`.